_events/2022-All-Hands-Meeting.md

```diff
@@ -24,7 +24,7 @@ Save the dates! The OSG All-Hands Meeting 2022 will be 14–18 March 2022. We ar
 Focus areas for 2022 will include:
 - State of OSG and vision for its future
 - New types of resources that support new types of applications
-- Research that benefitted from OSG
+- Research that benefited from OSG
```

Comment:
Huh, interesting, the Apple Dictionary says that either form is ok, but for me the double-t is required. Can we revert this one change? Otherwise, the rest looks good and thanks for catching all the typos above!